components/list/test/list.vdiff.js

```diff
@@ -380,11 +380,21 @@ describe('list', () => {
 
 	describe('controls', () => {
 		function createListWithControls(opts) {
-			const { color2, selectable, selected, selectAllPages } = { selectable: true, selected: [false, false], selectAllPages: false, ...opts };
+			const { actions, color2, extendSeparators, selectable, selected, selectAllPages } = {
+				actions: false,
+				extendSeparators: false,
+				selectable: true,
+				selected: [false, false],
+				selectAllPages: false,
+				...opts
+			};
 			return html`
-				<d2l-list item-count="${ifDefined(selectAllPages ? '50' : undefined)}" style="width: 400px;">
+				<d2l-list item-count="${ifDefined(selectAllPages ? '50' : undefined)}" ?extend-separators="${extendSeparators}" style="width: 400px;">
 					<d2l-list-controls slot="controls" ?no-selection="${!selectable}" ?select-all-pages-allowed="${selectAllPages}" no-sticky>
-						${selectable ? nothing : html`<d2l-button-subtle text="Action"></d2l-button-subtle>`}
+						${actions ? html`
+							<d2l-selection-action text="Delete" icon="tier1:delete"></d2l-selection-action>
+							<d2l-selection-action text="Edit" icon="tier1:edit"></d2l-selection-action>
+						` : nothing}
```

Comment:
For some reason this was rendering a subtle button (not a selection action) and only when things _weren't_ selectable... which seems like a weird case. I've kept the use case (test 1 down below) but am now rendering 2 actions.